output_dac.cpp

```diff
@@ -184,59 +183,65 @@ void AudioOutputAnalog::begin(void)
 	// commandeer FTM1 for timing (PWM on pin 3 & 4 will become 22 kHz)
 	FTM1_SC = 0;
 	FTM1_CNT = 0;
-	FTM1_MOD = (uint32_t)((F_PLL/2) / AUDIO_SAMPLE_RATE_EXACT + 0.5);
-	FTM1_SC = FTM_SC_CLKS(1);
+	FTM1_MOD = (uint32_t)((F_PLL/2) / 44117.64706/*AUDIO_SAMPLE_RATE_EXACT*/ + 0.5);
```

Comment:
This bit would maybe want to change - I was interested in playing back full 44 kHz audio, and in `AudioStream.h` Teensy-LC has been specified to use only 22 kHz.  I didn't see a specific reason not to use the higher rate for my project, so I did that, and I thought it might be good to have a discussion about it here.  But I have no particular attachment to the official library code running at 44 kHz vs 22.  Obviously we don't want the actual sample rate in code here, but the constant isn't part of the audio library, so changing it would be a larger discussion.